climada/util/coordinates.py

```diff
@@ -944,6 +944,35 @@ def country_iso_alpha2numeric(isos):
         nums.append(num)
     return nums[0] if return_int else nums
 
+def country_iso_numeric2alpha(isos_numeric):
+    """Convert ISO 3166-1 numeric-3codes to alpha-3
+    
+    Parameters
+    ----------
+    isos_numeric : int or list of int
+        ISO numeric-3 codes of countries (or single code).
+        
+    Returns
+    -------
+    isos : str or list of str
+        Will only return a list if the input is a list.
```

Comment:
What are the isos returned? This is needed in the description.

Reply:
Description has been added